src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -150,6 +151,40 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                             return;
                         }
                     }
+                } else if(args[2].equals("massadd")){
+                    if(args.length == 4) {
+                        Group group = getPermissions(channel).getGroup(groupString);
+                        if (group == null) {
+                            MessageUtils.sendErrorMessage("That group doesn't exist!!", channel);
+                            return;
+                        } else {
+                            List<Member> roleMembers;
+                            String roleName = "";
```

Comment:
wouldn't work cause people can send us ids or just the name. and even mentions don't work like that, It would only work with @here and @everyone